gerrit-entrypoint.sh

```diff
@@ -9,6 +9,14 @@ set_secure_config() {
   su-exec ${GERRIT_USER} git config -f "${GERRIT_SITE}/etc/secure.config" "$@"
 }
 
+wait_for_database() {
+  echo "Waiting for database connection $1:$2 ..."
+  until nc -z $1 $2; do
+    sleep 1
+  done
+  sleep 2
```

Comment:
Is there any reason `sleep 2` is necessary here after db connection has been confirmed?